### ISSUE TITLE
mobile menu

### ISSUE THREAD

**Issue by boyod** [siteorigin/vantage#268]
after the upgrade to the latest version, the mobile menu in live editor stopped working and the part of the localization is missing

**Comment by Misplon** [comment]
Hi @boyod

Thanks for reaching out. 

Page Builder is injecting `pointer-events: none;` into the menu link and the scroll to top link which is the cause of this in the Live Editor. I'll chat with the team and get back to you with feedback.

Which localization are you missing?

**Comment by boyod** [comment]
russian localization are  missing partially

**Comment by Misplon** [comment]
Are you referring to the theme's .pot file? Which strings, in particular, are missing?

**Comment by Misplon** [comment]
If strings are missing in https://themes.svn.wordpress.org/vantage/1.7.9/languages/vantage.pot, that's definitely something I can take a look at.

From the translation side, Vantage isn't completely translated into Russian, you can see the progress ,and contribute if you'd like to, here: https://translate.wordpress.org/projects/wp-themes/vantage/ru/default

**Comment by boyod** [comment]
not translated: "Edit History" "Theme settings>" "Theme Design>" "Page Template Settings>"

**Comment by Misplon** [comment]
Thanks again for posting. We rolled back to the previous version of Vantage and the mobile menu and back to top links also don't work in the Live Editor. I've chatted with our lead Page Builder developer, the reason these links are disabled is to prevent the user from accidentally navigating away from the page being worked on. Unlike the Customizer, the Live Editor is linked to the particular page you're editing. I hope that helps to explain what's happening.
